.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "useTabs": false,
+  "semi": true,
+  "trailingComma": "none",
```

Comment:
I do value trailing comma for git blame purposes, and I think it's also how Dredd is set up.
However, I've stumbled into issues when trailing commas in commonJS fail execution of a module in NodeJS.

Reply:
Dredd is set up to have trailing commas in objects/arrays, but not in functions. It was because of Node 6 support. Is is possible that the issues you've experienced were because of having trailing commas in function arguments and executing with Node 6?

https://github.com/apiaryio/dredd/blob/master/.eslintrc.js#L12:L20